### ISSUE TITLE
[A11y] drawer a11y issues

### ISSUE THREAD

**Issue by gbowne1** [gbowne1/codestream#27]
We have pretty good use of aria-expanded and visibility: hidden.  Several things stick out though as potential issues.

There should be a link the button to the drawer so a screen reader knows what is being expanded.

I think the fix here should be to add aria-controls="categoryDrawer" to the drawerToggle button.

Add role="navigation" and aria-label="Category Menu" to the drawer.

the js should not need any changes.

**Comment by djnadackal** [comment]
Shall I work on this?

BTW when I tried setting it up I am not able to run

<img width="541" height="157" alt="Image" src="https://github.com/user-attachments/assets/6c8d4e82-6a0e-4bcb-8ddc-dba6570fd807" />

**Comment by gbowne1** [comment]
You could try 

`chmod -R +x node_modules/.bin`

Then npm run dev.

If not remove the node_modules folder and package-lock.json then run npm install again 

as a last resort try npx vite

**Comment by djnadackal** [comment]
Shall I remove node_modules from the git respository?

since it shouldn't be normally added to be tracked

**Comment by djnadackal** [comment]
thanks I will delete and push with my changes.
